.github/workflows/release-docs.yml

```diff
@@ -0,0 +1,62 @@
+name: Release docs pipeline
+# Only trigger, when the release workflow has completed
+on:
+  workflow_run:
+    workflows: ["Release pipeline"]
+    types:
+      - completed
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: write
+  pages: write
+  id-token: write
+
+# Allow one concurrent deployment
+concurrency:
+  group: "pages"
+  cancel-in-progress: true
+
+jobs:
+  build-and-publish:
+    name: Create a new  docs release
+    runs-on: ubuntu-22.04
+    # runs only, when the workflow was succeeded
+    if: ${{ github.event.workflow_run.conclusion == 'success' }} 
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Set release tag
+        run: |
+          R=${GITHUB_REF#"refs/tags/"}
+          echo "RELEASE=$R" >> $GITHUB_ENV
+      
+      - name: Set up git author
+        run: |
+          remote_repo="https://${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git"
+          git config --global user.name "${GITHUB_ACTOR}"
+          git config --global user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+          git remote rm origin
+          git remote add origin "${remote_repo}"
+          git push --set-upstream origin master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Add new changelog file to mkdocs.yml navigation under Changelog section
+        run: |
+          /bin/bash ./update_mkdocs.sh
```

Comment:
Any reason why the script is in a separate file instead of putting it here?

Reply:
My only argument for adding it to the pipeline directly is so we have a cleaner repository. Usually, I see these scripts in `/hack` directory [(example)](https://github.com/kubernetes/kubernetes/tree/master/hack). So IMO, I would either put it into the pipeline directly or created `/hack` directory and store it there.

Reply:
I agree with Miro's suggestion to put it into the pipeline directly. Let's do that

Reply:
I have put the script inside `hack` directory. Guess we can keep it like this, if you don't have any objections.

Reply:
Sorry @MarioUhrik, I haven't seen your comment, when I was posting mine.
Going to put it inside `release-docs.yml` pipeline.

Reply:
All should be done